src/expr/src/explain/text.rs

```diff
@@ -1051,7 +1040,7 @@ pub trait HumanizerMode: Sized + Clone {
     /// This will produce a [`HumanizerMode`] instance that redacts output in
     /// production deployments, but not in debug builds and in CI.
     fn default() -> Self {
-        let redacted = !SOFT_ASSERTIONS.load(Ordering::Relaxed);
+        let redacted = SOFT_ASSERTIONS.load(Ordering::Relaxed);
```

Comment:
Could you explain why this needs to change? I thought soft assertions are not enabled in production.

Reply:
It doesn't need to change, I had commented the line out while developing and didn't revert this cleanly.

Reply:
Maybe we could factor out the many occurrences of `SOFT_ASSERTIONS.load(Ordering::Relaxed)` to a function in `mz_ore` or somewhere.

Reply:
I'll define an alias for `!SOFT_ASSERTIONS.load(Ordering::Relaxed)` that are used to govern `EXPLAIN` output in the next PR.